src/PrestaShopBundle/Form/Admin/Type/ButtonCollectionType.php

```diff
@@ -84,8 +86,11 @@ public function configureOptions(OptionsResolver $resolver)
                 'label' => false,
                 'buttons' => [],
                 'justify_content' => 'space-between',
+                'inline_buttons_limit' => null,
+                'force_labels' => false,
```

Comment:
Actually, I believe the initial usages of the `ButtonCollectionType` was inline buttons with labels. So the default option here should be reversed, the default behaviour is to have labels for inline buttons.